src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -532,11 +532,23 @@ def _create_pgm_input_transformers(self):
         parallel = self._get_pp_attr("trafo", "parallel", 1)
         sn_mva = self._get_pp_attr("trafo", "sn_mva")
         switch_states = self.get_switch_states("trafo")
-        winding_types = self.get_trafo_winding_types()
 
-        tap_side = self._get_pp_attr("trafo", "tap_side")
+        tap_side = self._get_pp_attr("trafo", "tap_side", np.nan)
         tap_nom = self._get_pp_attr("trafo", "tap_neutral", np.nan)
         tap_pos = self._get_pp_attr("trafo", "tap_pos", np.nan)
+        # Do not use taps when mandatory tap data is not available
+        no_taps = np.equal(tap_side, None) | np.isnan(tap_pos) | np.isnan(tap_nom)
+        tap_nom[no_taps] = 0
+        tap_pos[no_taps] = 0
+        tap_side[no_taps] = "hv"
+
+        winding_types = self.get_trafo_winding_types()
+        clocks = np.round(self._get_pp_attr("trafo", "shift_degree", 0.0) / 30) % 12
+        # Default vector group for odd clocks = DYn and for even clocks = YNyn
+        no_vector_groups = np.isnan(winding_types["winding_from"]) | np.isnan(winding_types["winding_to"])
+        no_vector_groups_dyn = no_vector_groups & (clocks % 2)
+        winding_types[no_vector_groups] = WindingType.wye_n
```

Comment:
Are you referring here to `winding_from` or `winding_to`?

Reply:
I set both, from and to windings to wye_n then only the odd clocks are set to delta